webservices/test_api.py

```diff
@@ -276,10 +276,6 @@ def test_committee_by_candidate(self):
         results =  self._results('http://localhost:5000/candidate/P60000247/committees?year=*')
         self.assertEquals(3, len(results))
 
-    def test_candidates_by_committee(self):
```

Comment:
`committee/id/candidates` was the same function as `candidate/id` and since we were un-nesting I took it out.  I would think that in the next step, when we add the candidate-committee linkages, they will live at `committee/id/candidates`. I thought that was going to be different enough that it will require a new test.